examples/shift_reg/default/sdkconfig.defaults

```diff
@@ -0,0 +1 @@
+# add required non-default option for the example if any
```

Comment:
Please delete this file if the example does not require special ESP-IDF settings.